src/utils/hooks/useCreateFinalDecisionReview.ts

```diff
@@ -12,16 +12,16 @@ import { useGetFullReviewStructureAsync } from './useGetReviewStructureForSectio
 type UseRemakePreviousReviewMutationReturnType = ReturnType<typeof useCreateReviewMutation>
 type PromiseReturnType = ReturnType<UseRemakePreviousReviewMutationReturnType[0]>
 // hook used to restart a review, , as per type definition below (returns promise that resolve with mutation result data)
-type UseRemakePreviousReview = (props: {
+type UseCreateFinalDecisionReview = (props: {
```

Comment:
Renaming the hook to be clearer what's doing

Reply:
So much clearer! :)